meta2v2/meta2_utils_lb.c

```diff
@@ -192,9 +192,8 @@ static gchar*
 m2v2_build_chunk_url_storage (const struct storage_policy_s *pol,
 		const gchar *str_id)
 {
+	(void)(str_id);
 	switch(data_security_get_type(storage_policy_get_data_security(pol))) {
-	case STGPOL_DS_BACKBLAZE:
-		return g_strconcat("b2/", str_id, NULL);
```

Comment:
Since there is only one choice, no need for a switch.

Reply:
:+1:

Reply:
By the way, the calling function will crash if this function returns NULL. I suggest removing this one, and modifying `generate_chunk_bean`.